game-core/src/main/java/games/strategy/util/CountUpAndDownLatch.java

```diff
@@ -10,16 +9,8 @@
  * Implements AQS behind the scenes similar to CountDownLatch.
  * Class is hobbled together from various learnings and tickets on stackexchange/stackoverflow.
  */
-public class CountUpAndDownLatch implements Serializable {
```

Comment:
The only usage of this class is inside ConcurrentOddsCalculator, a non-serializeable class